src/proposals/20201128/SparkEthereum_20241128.sol

```diff
@@ -0,0 +1,115 @@
+        uint256 _amount,
+        uint32 _minGasLimit,
+        bytes calldata _extraData
+    ) external;
+}
+
+/**
+ * @title  Nov 28, 2024 Spark Ethereum Proposal
+ * @notice Sparklend: update WBTC and cbBTC parameters
+ *         Morpho: onboard PT-USDe-27Mar2025 and increase PT-sUSDe-27Mar2025 cap
+ * @author Wonderland
+ * Forum:  https://forum.sky.money/t/28-nov-2024-proposed-changes-to-spark-for-upcoming-spell/25543/2
+ */
+contract SparkEthereum_20241128 is SparkPayloadEthereum {
+
+    address internal constant PT_27MAR2025_PRICE_FEED = 0x38d130cEe60CDa080A3b3aC94C79c34B6Fc919A7;
+    address internal constant PT_SUSDE_27MAR2025      = 0xE00bd3Df25fb187d6ABBB620b3dfd19839947b81;
+    address internal constant PT_USDE_27MAR2025       = 0x8A47b431A7D947c6a3ED6E42d501803615a97EAa;
+    address internal constant MORPHO                  = 0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb;
+    uint256 internal constant USDS_MINT_AMOUNT        = 90_000_000e18;
+
+    function collateralsUpdates() public pure override returns (IEngine.CollateralUpdate[] memory) {
+        IEngine.CollateralUpdate[] memory updates = new IEngine.CollateralUpdate[](2);
+
+        // Reduce LT from 65% to 60%
+        updates[0] = IEngine.CollateralUpdate({
+            asset:          Ethereum.WBTC,
+            ltv:            EngineFlags.KEEP_CURRENT,
+            liqThreshold:   60_00,
+            liqBonus:       EngineFlags.KEEP_CURRENT,
+            debtCeiling:    EngineFlags.KEEP_CURRENT,
+            liqProtocolFee: EngineFlags.KEEP_CURRENT,
+            eModeCategory:  EngineFlags.KEEP_CURRENT
+        });
+
+        // Increase LT from 70% to 75%
+        // Increase LTV from 65% to 74%
+        updates[1] = IEngine.CollateralUpdate({
+            asset:          Ethereum.CBBTC,
+            ltv:            74_00,
+            liqThreshold:   75_00,
+            liqBonus:       EngineFlags.KEEP_CURRENT,
+            debtCeiling:    EngineFlags.KEEP_CURRENT,
+            liqProtocolFee: EngineFlags.KEEP_CURRENT,
+            eModeCategory:  EngineFlags.KEEP_CURRENT
+        });
+
+        return updates;
+    }
+
+    function _postExecute() internal override {
+        // update existing cap for PT-sUSDe-27Mar2025 200m -> 400m
+        IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).submitCap(
+            MarketParams({
+                loanToken:       Ethereum.DAI,
+                collateralToken: PT_SUSDE_27MAR2025,
+                oracle:          PT_27MAR2025_PRICE_FEED,
+                irm:             Ethereum.MORPHO_DEFAULT_IRM,
+                lltv:            0.915e18
+            }),
+            400_000_000e18
+        );
+
+        MarketParams memory USDeMarket = MarketParams({
+            loanToken:       Ethereum.DAI,
+            collateralToken: PT_USDE_27MAR2025,
+            oracle:          PT_27MAR2025_PRICE_FEED,
+            irm:             Ethereum.MORPHO_DEFAULT_IRM,
+            lltv:            0.915e18
+        });
```

Comment:
Can't we onboard as before? 
https://github.com/marsfoundation/spark-spells/blob/master/archive/20241017/SparkEthereum_20241017.sol#L119-L129
In this case, we don't need to set supply cap separately

Reply:
From the forum post I initially interpreted both markets were supposed to use the same pricefeed and didn't know there was another one with a different pricefeed for USDE already created. I just pushed ace6e7b which uses this already-created market and has only one call to set the supply cap. Does that count as 'onboarding as before'? or am I missing something else?

Reply:
Sorry, I needed to specify what I meant. Now I realized what you've done.
I meant that `MarketParams` were defined inside `submitCap()` function. Something like
```
IMetaMorpho(Ethereum.MORPHO_VAULT_DAI_1).submitCap(
            MarketParams({
                loanToken:       Ethereum.DAI,
                collateralToken: PT_USDE_27MAR2025,
                oracle:          PT_USDE_27MAR2025_PRICE_FEED,
                irm:             Ethereum.MORPHO_DEFAULT_IRM,
                lltv:            0.915e18
            }),
            100_000_000e18
        );
```
Anyway `USDeMarket` isn't used even two times. Just to make coding style more consistent